### ISSUE TITLE
Clean up for mWeb Download App Banner

### ISSUE THREAD

**Issue by grgia** [Expensify/App#25820]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

Feature Added here - https://github.com/Expensify/App/pull/25418

Clean up points:

- Remove blue outline on mWeb
- Add AppIcon to Expensicons
- Change shouldStackButtons to use Flex order 
- fix `downloadAppModal` key capitalization in en.js / es.js 
- if the banner is dismissed it doesn't reappear on sign-out - only after refresh (only affects Android Chrome).

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~0170b9f8d80a16cdaf</li>
        <li>Upwork Job ID: 1694645507289415680</li>
        <li>Last Price Increase: 2023-08-24</li>
    </ul>
</details>

**Comment by grgia** [comment]
@mananjadhav sorry for the ping, assigning @jjcoffee as they have context from the original PR!

**Comment by dylanexpensify** [comment]
@grgia mind if we get an update on this?

**Comment by grgia** [comment]
@dylanexpensify this is code clean up and lower priority. Do we have an issue for replacing this with smart banners?

**Comment by dylanexpensify** [comment]
Done
